pywinauto/controls/common_controls.py

```diff
@@ -1384,7 +1384,7 @@ def select(self):
             self.elem)                    # item to select
 
         if retval != win32defines.TRUE:
-            raise ctypes.WinError()
+            raise ctypes.WinError()

```

Comment:
What are the changes up to this point in this file?
Line breaks?

Reply:
Maybe yes. This is weird, because I have Git option `autocrlf = true` in my settings. I've re-created this PR to avoid much larger diffs for some files. Locally I see only CR-LF line endings everywhere and for these lines particularly. In the repo it should be all `LF`s but it's not easy to check. So we may have the situation when this commit fixes or breaks these `LF`s. I think this is not principal for Windows specific code. Especially because I'm preparing release package on Windows. :)